src/app/modules/item/components/item-permissions/item-permissions.component.html

```diff
@@ -93,11 +93,13 @@
             ></button>
           </div>
 
+          <!-- NOTE: For now, we can assume that giver permissions is equivalent to user permissions on item. -->
```

Comment:
I am very sorry, I've completely misunderstood your initial question. 

> From my understanding, in this context (watching a group) giverPermissions=item.permissions,

Yes and that's all. 

All my remarks were about the permission source group... but that's completely unrelated and another issue.

So this comment you added does not make sense. Can you remove it please? 😔

Reply:
No pb, thanks for taking the time to check :pray: 
Fixed